SlackChatbot/Readme

```diff
@@ -0,0 +1,13 @@
+Activate virtual env : source venv/bin/activate
+Start server : python slack_bot.py
+Check if server is running by going to this URL http://127.0.0.1:3000/
+
+https://slack.com/oauth/v2/authorize?client_id=7855184757495.7892528086800&scope=chat:write,channels:read,groups:read&redirect_uri=https://ripe-rooms-own.loca.lt/oauth/callback
```

Comment:
Is this url universal? Otherwise you may want to remove and include instructions on obtaining this link.